app/controllers/home_controller.rb

```diff
@@ -1,8 +1,12 @@
 class HomeController < ApplicationController
+  
   def index 
     @products = ProductFilterService.new(params).filter
-  
+    
```

Comment:
You also have a space here, not just the newline. Rubocop would complain about this